data/prod/news/articles/c9rpqy7pmypo.json

```diff
@@ -1,877 +1,923 @@
-          "thingId": "b2412496-55d2-43ab-9a76-d205428090e2",
-          "thingLabel": "Diana, Princess of Wales",
-          "thingType": ["Thing", "Person"]
-        },
-        {
-          "thingUri": "http://www.bbc.co.uk/things/1efbf3e5-b330-49a1-b531-b507ab027c96#id",
-          "thingId": "1efbf3e5-b330-49a1-b531-b507ab027c96",
-          "thingLabel": "Queen Victoria",
-          "thingType": ["Person", "Thing"]
-        },
-        {
-          "thingUri": "http://www.bbc.co.uk/things/ec71caf7-c451-4ab8-8a29-70582347a42c#id",
-          "topicId": "cz4pr2gd414t",
-          "topicName": "Duchess of Cambridge",
-          "curationType": ["vivo-stream"],
-          "thingId": "ec71caf7-c451-4ab8-8a29-70582347a42c",
-          "thingLabel": "Duchess of Cambridge",
-          "thingType": ["Thing", "Person"]
-        }
-      ]
-    }
-  },
   "content": {
     "model": {
       "blocks": [
         {
-          "type": "headline",
           "model": {
             "blocks": [
               {
-                "type": "text",
                 "model": {
                   "blocks": [
                     {
-                      "type": "paragraph",
                       "model": {
-                        "text": "Royal wedding 2018: Bouquet laid on tomb of unknown warrior",
                         "blocks": [
                           {
-                            "type": "fragment",
                             "model": {
-                              "text": "Royal wedding 2018: Bouquet laid on tomb of unknown warrior",
-                              "attributes": []
-                            }
+                              "attributes": [],
+                              "text": "Royal wedding 2018: Bouquet laid on tomb of unknown warrior"
+                            },
+                            "type": "fragment"
                           }
-                        ]
-                      }
+                        ],
+                        "text": "Royal wedding 2018: Bouquet laid on tomb of unknown warrior"
+                      },
+                      "type": "paragraph"
                     }
                   ]
-                }
+                },
+                "type": "text"
               }
             ]
-          }
+          },
+          "type": "headline"
         },
         {
-          "type": "image",
           "model": {
             "blocks": [
               {
-                "type": "rawImage",
                 "model": {
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "A933/production/_101651334_bouquet_pa.jpg",
-                  "originCode": "cpsprodpb",
-                  "copyrightHolder": "PA"
-                }
+                  "copyrightHolder": "PA",
+                  "height": 360,
+                  "locator": "0a88/test/5fe2b600-ccaa-11e8-b0bf-f33155223fc4.jpg",
+                  "originCode": "cpsdevpb",
```

Comment:
Just want to make absolutely sure we've updated from the correct env - this looks like test data...

Reply:
These assets dont exist on 'prod', theyre test assets. We only have them in 'prod' as we didnt have live assets before. Leaving them here though as theyre useful locally.